IS7024/HBowl.cs

```diff
@@ -207,7 +207,7 @@ public enum VenueDisplayName { HollywoodBowl };
 
     public enum CountryDisplayName { Ca, Us };
 
-    public enum MetroAreaDisplayName { LosAngelesLa };
+    public enum MetroAreaDisplayName { LosAngeles};
```

Comment:
This change was made so that "LosAngeles" shows up when the "DisplayName" is called for MetroArea.